docs/content/contributing/issues-and-pull-requests.md

```diff
@@ -37,6 +37,28 @@ The development workflow is the following:
 5. Document your improvements if necessary (e.g. if you introduced new environment variables, describe those in the [ENV documentation][docs-environment]) and add your changes the changelog under the "Unreleased" section
 6. [Commit][commit] (and [sign your commit][gpg]), push and create a pull-request to merge into `master`. Please **use the pull-request template** to provide a minimum of contextual information and make sure to meet the requirements of the checklist.
 
+```mermaid
+%%{init: { 'gitGraph': {'mainBranchName': 'master', 'showCommitLabel': false }}}%%
+gitGraph
+    commit
+    branch your-branch
+    checkout your-branch
+    commit
+    checkout master
+    commit
+    branch another-branch
+    checkout another-branch
+    commit
+    commit
+    checkout master
+    merge another-branch
+    checkout your-branch
+    merge master
+    commit
+    checkout master
+    merge your-branch
+```
+
```

Comment:
Oh, may need to remove this again. Does not actually belong to changes in this PR. Reminder to future-self.

Reply:
Done.